vms/volunteer/models.py

```diff
@@ -35,14 +35,16 @@ class Volunteer(models.Model):
             ),
         ],
     )
-    city = models.CharField(
-        max_length=75,
-        validators=[
-            RegexValidator(
-                r'^[(A-Z)|(a-z)|(\s)|(\-)]+$',
-            ),
-        ],
-    )
+    #city = models.CharField(
+        #max_length=75,
+        #validators=[
+            #RegexValidator(
+                #r'^[(A-Z)|(a-z)|(\s)|(\-)]+$',

```

Comment:
after removing the validator, we won't be allowing user to add a new country, is it?